.github/workflows/valkey.yml

```
@@ -59,15 +82,88 @@ jobs:
+          RUN_SLOW_TESTS_TOO=1 tini -s -w hatch -- run test:cov --durations=5
 
       - name: Upload coverage to Codecov
         uses: codecov/codecov-action@v6.0.1
         with:
           files: ./coverage.xml
           fail_ci_if_error: false
           token: ${{ secrets.CODECOV_TOKEN }} # Store your Codecov token in secrets
+
+  cluster-valkey:
+    name:
+      "Cluster/Py${{ matrix.python-version }}/Valkey${{ matrix.valkey-version }}\
```

actionlint (1.7.12)

[error] property "valkey-variant" is not defined in object type {python-version: number; redis-py-version: number; valkey-version: number; variant: string}

(expression)